docs/reference/pyboard.py.rst

```diff
@@ -0,0 +1,128 @@
+.. _pyboard_py:
+
+The pyboard.py tool
+===================
+
+This is a standalone Python tool that runs on your PC that provides a way to:
+
+* Quickly run a Python script or command on a MicroPython device. This is useful
+* while developing MicroPython programs to quickly test code without needing to
+* copy files to/from the device.
```

Comment:
Does it need the stars on each line of the bullet point?  Or is this a more robust way to define bullet points?

Reply:
No that's just my editor failing at understanding how to wrap reStructuredText. Fixed.